sphinx_rtd_theme/layout.html

```diff
@@ -207,6 +207,20 @@
   </script>
   {% endif %}
 
+  {% if theme_analytics_id %}
+  <!-- Theme Analytics -->
+  <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', '{{ theme_analytics_id }}', 'auto');
+  ga('send', 'pageview');
+  </script>
+
+  {% endif %}
```

Comment:
I think you're conflating the RTD theme with RTD -- lots of people use this theme outside of our implementation, so we shouldn't be handicapping those folks. Having a standard override mechanism gives us _more_ control over how people are integrating GA. The current status quo has them overriding templates, and there's no way for us to unset it. This way if they're using the standard mechanism, we can *unset* it from RTD's side, and avoid loading it twice. This seems like a win/win, instead of telling people to override templates to do it.